src/lib/tecs/TECS.cpp

```diff
@@ -778,6 +795,9 @@ void TECS::update(float pitch, float altitude, float hgt_setpoint, float EAS_set
 		_detect_uncommanded_descent(throttle_setpoint_max, altitude, hgt_setpoint, equivalent_airspeed * eas_to_tas,
 					    control_setpoint.tas_setpoint);
 
+		// Detect stall
+		_detect_airspeedless_stall(dt, hgt_setpoint, altitude, pitch, hgt_rate);
```

Comment:
@KonradRudin I was wondering if it would make more sense to use the states from the reference model instead of the raw setpoints. Not sure though, any opinion?

Reply:
We're only talking about the hgt_setpoint, right? I think it does not matter much. I just wanted to make sure that we see that the original intend was to climb. Also one could think of alter the commanded setpoint before the reference model, whereas currently the setpoints are altered afterwards, then the detection would not work anymore.